public/Day 125/index.html

```
@@ -0,0 +1,469 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ meta ] must be self closed.

(tag-self-close)